packages/components/src/components/tooltip/tooltip.ts

```diff
@@ -256,40 +253,94 @@ export default class SdTooltip extends SolidElement {
   }
 
   render() {
+    const isStart = this.placement.endsWith('-start');
+    const isEnd = this.placement.endsWith('-end');
+
+    const skiddingMap = new Map([
+      ['start', '2'],
+      ['end', '-2']
+    ]);
+
+    const isSm = this.size === 'sm';
     return html`
       <sd-popup
         part="base"
+        id="tooltip"
         exportparts="
           popup:base__popup,
           arrow:base__arrow
         "
-        class=${classMap({
-      tooltip: true,
-      'tooltip--open': this.open
-    })}
+        class=${cx('tooltip', this.open && 'tooltip--open')}
         placement=${this.placement}
-        distance=${this.distance}
-        skidding=${this.skidding}
+        distance="12"
+        skidding=${((skiddingMap.get(isStart ? 'start' : isEnd ? 'end' : 'default') || 0) as number) * (isSm ? -1 : 1)}
         strategy=${this.hoist ? 'fixed' : 'absolute'}
         flip
         shift
         arrow
+        arrow-padding="0"
+        auto-size="vertical"
       >
-        <slot slot="anchor" aria-describedby="tooltip"></slot>
+        <slot slot="anchor" class=${cx(this.size === 'lg' ? 'text-xl' : 'text-base')}>
+          <sd-icon library="system" name="info-sign" class="sd-interactive rounded-full" tabindex="0"></sd-icon>
+        </slot>
 
         <slot
           name="content"
           part="body"
-          id="tooltip"
-          class="tooltip__body"
-          role="tooltip"
+          id="tooltip-body"
+          class="tooltip__body bg-primary text-white py-3 px-4 block shadow rounded-none text-sm text-left"
+          role="tooltiprole"
           aria-live=${this.open ? 'polite' : 'off'}
+          tabindex="0"
         >
           ${this.content}
         </slot>
       </sd-popup>
     `;
   }
+  static styles = [
```

Comment:
I think you can now change most of the plain css styles to tailwind classes
for example: 
```
#tooltip[placement^='top']::part(popup) {
    @apply origin-bottom;
}
```